src/Symfony/Bundle/FrameworkBundle/Resources/config/notifier_transports.php

```diff
@@ -135,6 +136,10 @@
             ->parent('notifier.transport_factory.abstract')
             ->tag('texter.transport_factory')
 
+        ->set('notifier.transport_factory.gitter', GitterTransportFactory::class)
+            ->parent('notifier.transport_factory.abstract')
+            ->tag('texter.transport_factory')
```

Comment:
```suggestion
            ->tag('chatter.transport_factory')
```
as this transport uses the `ChatMessage` and not the `SmsMessage`, right?